examples/mqtt-service-ssl/mqtt_service_ssl.md

```diff
@@ -0,0 +1,18 @@
+# MQTT service - SSL/TLS
+
+The `mqtt:Service` receives messages from the MQTT server using the `mqtt:Listener` via SSL/TLS. SSL/TLS can be enabled by configuring the `secureSocket`, which requires a certificate or a truststore. Further, mTLS can be enabled by providing a certificate and private key of the service or a keystore. Use this to connect to an MQTT server secured with SSL.
```

Comment:
```suggestion
The `mqtt:Service` receives messages from the MQTT server using the `mqtt:Listener` via SSL/TLS. SSL/TLS can be enabled by configuring the `secureSocket`, which requires a certificate or a trust store. Further, Mutual TLS (mTLS) can be enabled by providing a certificate and private key of the service or a keystore. This can be used to connect to an MQTT server secured with SSL.
```